### ISSUE TITLE
Activity colouring depending on HEC usage

### ISSUE THREAD

**Issue by varosi** [haskell/ThreadScope#82]
Intel VTune have similar profiling information which is helping to find places where not all available cores (HECs) are used. 
The idea is to use tri-color/semaphore (red, yellow, green) to paint the activity diagram depending on HEC usage. For example, if activity is less than 50% it's red, less than 75% yellow and more - green. So programs are pushed toward more parallel computations.

Actual HEC activity could be painted, too.

**Comment by Mikolaj** [comment]
I don't remember if it's enabled by default, but doesn't the graph where the height of stuff depends on how many HECs are used suffice?

**Comment by varosi** [comment]
Yes, it is, but not obvious enough in my opinion. It'll guide more into the best direction.

btw, other way of representing this is to paint the background only in three colors.
